lib/lightning_web/live/workflow_live/ai_assistant_component.ex

```diff
@@ -455,14 +522,22 @@ defmodule LightningWeb.WorkflowLive.AiAssistantComponent do
             id={"message-#{message.id}"}
             class="mr-auto p-2 rounded-lg break-words text-wrap flex flex-row gap-x-2 makeup-html"
           >
-            <div class="">
+            <div>
               <div class="rounded-full p-2 bg-indigo-200 text-indigo-700 ring-4 ring-white">
                 <.icon name="hero-cpu-chip" class="" />
               </div>
             </div>
 
             <div>
-              <%= message.content |> Earmark.as_html!() |> raw() %>
+              <div>
+                <%= message.content |> Earmark.as_html!() |> raw() %>
+              </div>
+              <div
+                class="flex mt-1 text-xs text-gray-400 select-none"
+                title="This message was generated using Claude, an LLM by Anthropic, and has not been verified by human experts"
+              >
+                This message was generated by an algorithm
```

Comment:
I think we should be upfront with calling it an AI. "This is an AI-generated message" reads better to me.

Reply:
Well it's clearly an AI-generated message. There's no benefit in calling that out.

People trust AIs, which can be a problem. They're a bit warier of algorithms - even though AI is just an algorithm. Silly really. But I think this is a good, subtle way of just setting a tiny distance between user and AI.

I prefer it even to "the AI can make mistakes", because mistakes aren't even necessarily what I'm worried about.

Reply:
Well.. I am not sold on that algorithm vs AI logic. We have done a fair job of talking about the risks and other "reasons-to-not-use" points for the AI feature. Stating that the response is AI-generated should not be a problem. Calling it an Algorithm doesn't fix it.

Reply:
Ok: so I'm gonna remove this warning message altogether from this PR.

I will raise a new PR with `Read, Review Verify` as the message, and make it a hyperlink to the docs site which explains what it means. That needs a docs PR to go with it though so it'll have to come later.